papermill/translators.py

```diff
@@ -313,6 +313,43 @@ def assign(cls, name, str_val):
         return 'var {} = {};'.format(name, str_val)
 
 
+class FSharpTranslator(Translator) :
+
+    @classmethod
+    def translate_none(cls, val) :
+        return 'None'
+
+    @classmethod
+    def translate_bool(cls, val) :
+        return 'true' if val else 'false'
+
+    @classmethod
+    def translate_int(cls, val):
+        strval = cls.translate_raw_str(val)
+        return strval + "L" if (val > 2147483647 or val < -2147483648) else strval
+
+    @classmethod
+    def translate_dict(cls, val):
+        tuples = '; '.join(
+            ["({}, {} :> IComparable)".format(cls.translate_str(k), cls.translate(v))
```

Comment:
Haven't done much F#, so not 100% sure this is the right / best option but I think it's a reasonable and working pattern.

Reply:
Yeah, I'm pretty sure this is sub-elegant!